infra/monitor/KalturaMonitorClient.php

```diff
@@ -805,4 +805,40 @@ public static function sendErrorEvent($errorCode)
 		
 		self::writeDeferredEvent($data);
 	}
+	
+	public static function checkApiRateLimit()
+	{
+		$uri = isset($_SERVER["REQUEST_URI"]) ? $_SERVER["REQUEST_URI"] : '';
+		if(!$uri)
+		{
+			return;
+		}
+		
+		preg_match('#/p/([^/]+)/sp/([^/]+)#', $uri, $matches);
+		if(!$matches || !count($matches))
+		{
+			return;
+		}
+		
+		$p = isset($matches[1]) ? $matches[1] : (isset($matches[2]) ? $matches[2] : null);
+		if(!$p)
+		{
```

Comment:
If partnerId comes from /sp/ - dont we need to divide by 100 to get the correct value?

Reply:
You're right, fixed.